x/gov/keeper/common_test.go

```diff
@@ -39,14 +41,18 @@ var (
 
 // getTestProposal creates and returns a test proposal message.
 func getTestProposal() []sdk.Msg {
-	legacyProposalMsg, err := v1.NewLegacyContent(v1beta1.NewTextProposal("Title", "description"), authtypes.NewModuleAddress(types.ModuleName).String())
+	legacyProposalMsg1, err := v1.NewLegacyContent(v1beta1.NewTextProposal("Title1", "description1"), authtypes.NewModuleAddress(types.ModuleName).String())
+	if err != nil {
+		panic(err)
+	}
+	legacyProposalMsg2, err := v1.NewLegacyContent(v1beta1.NewTextProposal("Title2", "description2"), authtypes.NewModuleAddress(types.ModuleName).String())
 	if err != nil {
 		panic(err)
 	}
 
 	return []sdk.Msg{
-		banktypes.NewMsgSend(govAcct, addr, sdk.NewCoins(sdk.NewCoin("stake", math.NewInt(1000)))),
```

Comment:
As we'll be accepting "update param" messages, wouldn't it make the test more "interesting" if we changed this `banktypes.NewMsgSend` by an example of "update param" message?

Reply:
I decided to leave only legacy proposals there to reduce the impact on tests using `getTestProposal()`
Instead, I tested all the update params types (also some unsupported) [here](https://github.com/0xPolygon/cosmos-sdk/pull/7/files#diff-c4662f2ddbaa109d151504c0064dc2856b323e1190fe0e149e8738fbc763463aR291). WDYT?

Reply:
Yes, makes sense to me. Resolved